test/compatibility/compatibility_test.py

```diff
@@ -84,9 +84,7 @@ def prepare_wt_branch(self, branch):
         # support older branches, which use autoconf.
         if not os.path.exists(os.path.join(build_path, 'build.ninja')):
             os.mkdir(build_path)
-            # Disable WT_STANDALONE_BUILD, because it is not compatible with branches 6.0 and
-            # earlier.
-            cmake_args = '-DWT_STANDALONE_BUILD=0 -DENABLE_PYTHON=1'
+            cmake_args = '-DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v4_clang.cmake -DENABLE_PYTHON=1'
```

Comment:
I understand this change is for enabling the fast truncate functionality to test it.

Reply:
Yes, that's right. I think I'm going to refactor this so that the `cmake_args` is passed into the function. I spoke to Peter this morning and he's planning some changes in the test/compatibility infrastructure, so this would be a temporary workaround.